lib/chef_zero/server.rb

```diff
@@ -103,14 +104,15 @@ class Server
       '/version',
     ]
 
-    def initialize(options = {})
-      @options = DEFAULT_OPTIONS.merge(options)
-      if @options[:single_org] && !@options.has_key?(:osc_compat)
-        @options[:osc_compat] = true
+    def initialize(**options)
```

Comment:
Is there a reason the arity of `options` changed? L-108 looks like it's expecting `options` to be a Hash.